skada/base.py

```diff
@@ -119,6 +119,25 @@ def transform(
         )
 
 
+class DAEstimator(BaseEstimator):
+    """ Generic DA estimator class
+
+    """
+
+    __metadata_request__fit = {'sample_domain': True}
+    __metadata_request__transform = {'sample_domain': True, 'allow_source': True}
```

Comment:
This should be `predict`, right? The estimator doesn't provide `transform` without a special mixin